db/schema.rb

```diff
@@ -26,4 +26,11 @@
     t.integer  "hire_count"
   end
 
+  create_table "users", force: :cascade do |t|
+    t.string   "email"
+    t.string   "password_digest"
+    t.integer  "role",            default: 0
+    t.datetime "created_at",                  null: false
+    t.datetime "updated_at",                  null: false
+  end
```

Comment:
I agree you made the right choice here. The flexibility that this offers will make it easier to build out the app later on, with minimal pain.